### DISCUSSION TITLE
Filtering IP addresses

### DISCUSSION THREAD

**Discussion by Mcdimmy007** [requarks/wiki#7318] [Help / Questions] [answered]
Hello,

I need help with setting up my wiki webpage to only be accessible from specified IP addresses. I have tried several methods using "allow" and "deny" in nginx. I also tried using "geo" tags but none seem to be working. 

I will appreciate any help I can get. Thank you.

**Comment by NGPixel** [comment, marked as answer]
That's not really a Wiki.js question. I suggest asking in stackoverflow nginx?

**Reply by Mcdimmy007** [reply]
Will do! Thanks
